cpp/log/cluster_state_controller-inl.h

```diff
@@ -9,12 +9,22 @@
 #include "fetcher/peer.h"
 #include "log/database.h"
 #include "log/etcd_consistent_store-inl.h"
+#include "monitoring/monitoring.h"
 #include "proto/ct.pb.h"
 
+using cert_trans::Gauge;
```

Comment:
But since you're using it from inside a ```namespace cert_trans```, you might not need the ```using``` statement anyway?